pennylane/ops/qubit/state_preparation.py

```diff
@@ -149,10 +158,12 @@ class StatePrep(StatePrepBase):
         as :math:`U|0\rangle = |\psi\rangle`
 
     Args:
-        state (array[complex]): a state vector of size 2**len(wires)
+        state (array[complex]): the state vector to prepare
         wires (Sequence[int] or int): the wire(s) the operation acts on
+        pad_with (float or complex): the value to pad the state vector with
```

Comment:
```suggestion
        pad_with (float | complex | None): the value to pad the state vector with
```

Reply:
Please do not upgrade the UI in this PR, we can have a follow-up PR for all these upgrades, that should be approved by Product.

Reply:
Also, why the original description from AmplitudeEmbedding is changed? It states:

pad_with (float or complex): if not None, the input is padded with this constant to size :math:`2^n`

Reply:
updated 👍